packages/bits/src/lib/wizard-v2/types.ts

```diff
@@ -1,22 +1,17 @@
 
+export type WizardIconStates = "active" | "visited" | "initial" | "error" | string;
+export type WizardStepStateConfig = Record<WizardIconStates, IWizardStepStateConfig>;
+
 export interface IWizardState {
     finished: boolean;
 }
 
 export interface IWizardConfig {
-    stepStateIcons: IWizardStepStateIconConfig;
+    stepState: WizardStepStateConfig;
     [key: string]: any;
 }
 
-export interface IWizardStepStateIconConfig {
-    icons?: Partial<IWizardIconStates>;
-    colors?: Partial<IWizardIconStates>;
-}
-
-export interface IWizardIconStates {
-    active: string;
-    visited: string;
-    initial: string;
-    error: string;
-    [key: string]: string;
+export interface IWizardStepStateConfig {
+    icon?: string;
+    color?: string;
 }
```

Comment:
I was thinking it would look like this:

```
export interface IWizardStepStateConfig {
    icon: IIconConfig;
}

export interface IIconConfig{
    icon?: string;
    color?: string;
}
```

That way it's obvious what the color property if for. But, I would settle for this too.

```
export interface IWizardStepStateConfig {
    icon?: string;
    iconColor?: string;
}
```